src/Veldrid.Tests/TestUtils.cs

```diff
@@ -229,7 +229,13 @@ protected Texture GetReadback(Texture texture)
                 Texture readback = RF.CreateTexture(ref desc);
                 CommandList cl = RF.CreateCommandList();
                 cl.Begin();
-                cl.CopyTexture(texture, readback);
+
+                //todo: temporary workaround due to cl.CopyTexture(texture, readback) not copying data properly for cubemaps in vulkan {
+                for (uint mip = 0; mip < texture.MipLevels; mip++)
```

Comment:
revert this; there was a fix in the Vulkan copy code that makes this workaround redundant.

Reply:
Done